cmd/earthly/buildkit.go

```diff
@@ -79,9 +79,21 @@ func (app *earthlyApp) configureSatellite(cliCtx *cli.Context, cloudClient cloud
 
 	app.console.Warnf("") // newline
 	app.console.Warnf("The following feature flags are recommended for use with Satellites and will be auto-enabled:")
-	app.console.Warnf("--new-platform, --use-registry-for-with-docker")
+	app.console.Warnf("  --new-platform, --use-registry-for-with-docker")
 	app.console.Warnf("") // newline
 
+	if app.ci {
+		app.console.Warnf("When using Satellites, the --ci flag is an alias for:")
+		app.console.Warnf("  --strict --no-output when running earthly in target mode.")
+		app.console.Warnf("  --strict when running earthly in artifact or image mode.")
+		app.console.Warnf("") // newline
```

Comment:
```suggestion
		app.console.Warnf("Please note that --use-inline-cache and --save-inline-cache are currently disabled when using Satellites and --ci")
		app.console.Warnf("") // newline
```

This wording is a bit easier to understand